### PR TITLE
Removed user id and password from login response

### PR DESCRIPTION
**Pull request by bisonlou**
### What does this PR do
- Removed the user Id from the login response
- Removed the password from the login response
- Formated user_model file to PEP8 standard-

### CODE REVIEW

**Comment by bisonlou** [comment]
I also changed the phone number expected datatype from int to string @kalsmic

**Review by PatrickMugayaJoel — COMMENTED** [review]
Looks good. We shall get there one step at a time.
With time the 'debug=true' in 'run_server.py' file will go to the config.py file or be set in environment

**Comment by kalsmic** [comment]
Well done @bisonlou, looking forward to the tests

**Comment by bisonlou** [comment]
Could only add succesful login and register tests because ther is no validation at the moment

File: app/views/incident_veiws.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 class CreateIncident(MethodView):
     @token_required
-    def post(current_user, self):
+    def post(self, current_user):
```
Comment (kalsmic) [inline app/views/incident_veiws.py:11]: 👍

**Review by kalsmic — APPROVED** [review]
Thanks, @bisonlou, You have covered the tasks as laid out in the user story.

**Pull request merged**